### DISCUSSION TITLE
Using "OR" condition on two different token attributes

### DISCUSSION THREAD

**Discussion by syrinecheriaa** [explosion/spaCy#12891] [Help: Other Questions] [answered]
I am adding a banch of spacy [matchers](https://spacy.io/api/matcher) to my pipeline ("en_core_we_sm"). 
To  each matcher I am adding a group of patterns. I know that each part (token_spec) of those pattern is applied on a single token. I want to know if one part of the pattern can contain an "OR" condition that is applied to two different attributes of the token. For example I want to catch token that
- have the lemma : "colleague"  **OR**  have its lower in ["friends", "enemies"]

For example I can have those to two patterns:
````
[{'LOWER': 'we'}, {'LOWER': 'are'}, {'LOWER': {"IN": ["friends","enemies" ]}},{"LOWER":"today"}]
[{'LOWER': 'we'}, {'LOWER': 'are'}, {'LEMMA': 'colleague'},{"LOWER":"today"}]
````
and I would like to write them like this:
````
[{'LOWER': 'we'}, {'LOWER': 'are'}, {'LEMMA': 'colleague', 'LOWER': {"IN": ["friends","enemies" ]}, "OP":"OR"}, {"LOWER":"today"}]
````

I would like to know if there is a way to work around this.
ps: I can not use the OP: "?" because  I don"t want to catch  "We are friends colleagues today".

**Comment by danieldk** [comment, marked as answer]
_OR_-patterns are currently not supported. So we recommend making two separate patterns instead.
